src/Microsoft.Net.Http.Headers/RangeHeaderValue.cs

```diff
@@ -33,7 +33,7 @@ public string Unit
             get { return _unit; }
             set
             {
-                HeaderUtilities.CheckValidToken(value, "value");
+                HeaderUtilities.CheckValidToken(value, nameof(_unit));
```

Comment:
Just glancing over this and not very familiar with project but this looks like it will change between the original string `"value"` and `nameof(_unit)` - is it an issue?